sirepo/pkcli/admin.py

```diff
@@ -99,13 +99,12 @@ def setup_dev_proprietary_code(sim_type, rpm_url):
       sim_type (str): simulation type
       rpm_url (str): Url of the rpm (file:// or http://)
     """
-    import sirepo.pkcli.admin
     import urllib.request
 
     assert pkconfig.channel_in('dev'), \
         'Only to be used in dev. channel={}'.format(pkconfig.cfg.channel)
 
-    d = sirepo.pkcli.admin.proprietary_code_dir(sim_type)
```

Comment:
Geez, who wrote that code!

Reply:
As Buffett said "I've made so many mistakes, if I tried to kick myself, my legs would be exhausted."